recursio-client/Characters/Player.gd

```diff
@@ -14,6 +14,11 @@ onready var _button_overlay: ButtonOverlay = get_node("KinematicBody/ButtonOverl
 onready var _aim_visuals = get_node("KinematicBody/AimVisuals")
 onready var _audio_player: AudioStreamPlayer = get_node("AudioStreamPlayer")
 
+
```

Comment:
defining this here is kinda wonky, shouldnt the be defined where ever `show_peperation_hud `is called?

Reply:
whoops I meant to comment the line below

Reply:
That would be in CharacterManager. So you're saying I should move this variable to the CharacterManager? `_hide_player_button_overlay` then?

Reply:
Not sure, just seems weird that `show_peparation_hud `can be called and then do nothing because an bool property was set to false on the object. Seems pretty messy.

Reply:
Anyway it's not that important. Just leave it for now. ^^